recipes/freetype-py/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "freetype-py" %}
+{% set version = "2.0.0.post6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 23c9dcd9c1797507b0e083c18848d476c72128137ca485665d149ce231001069
+
+build:
+  noarch: python
+  number: 0
+  # Note: --no-deps is currently required due to https://github.com/conda/conda-build/issues/3254
+  # Once resolved, it should be removed.
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools_scm
+  run:
+    - python
+    - freetype
+
+test:
+  imports:
+    - freetype
+
+about:
+  home: https://github.com/rougier/freetype-py
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Python binding for the freetype library'
+  description: |
+    Freetype Python provides bindings for the FreeType library.
+    Only the high-level API is bound.
+  doc_url: https://freetype-py.readthedocs.io/
+  dev_url: https://github.com/rougier/freetype-py
+
+extra:
+  recipe-maintainers:
+    - hoechenberger
+    - kastman
```

Comment:
@kastman - are you OK with being a maintainer here?

Reply:
@scopatz Yep - I’m here for that. Thanks!